openfe/tests/conftest.py

```diff
@@ -12,30 +12,62 @@
 from gufe import SmallMoleculeComponent, LigandAtomMapping
 
 
+class SlowTest:
+    """Plugin for a fixture that skips slow tests"""
+
+    def __init__(self, config):
+        self.config = config
+
+    def pytest_collection_modifyitems(self, items, config):
+        if (config.getoption('--runslow') or
+            os.getenv("OFE_SLOW_TESTS", default="false").lower() == 'true'):
+            return
+
+        msg = ("need --runslow pytest cli option or the environment variable "
+           "`OFE_SLOW_TESTS` set to `True` to run")
+        skip_slow = pytest.mark.skip(reason=msg)
+        for item in items:
+            if "slow" in item.keywords:
+                item.add_marker(skip_slow)
+
+
+class IntegrationTest:
+    """Plugin for a fixture that skips very slow integration tests"""
```

Comment:
these docs strings don't tell me how to use this.  something like `for tests marked with @pytest.mark.integration`

Reply:
Much docstring was added 🙃